packages/terra-dialog/src/Dialog.module.scss

```
@@ -32,9 +32,11 @@
     background: var(--terra-dialog-header-close-background, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(17, 17, 17, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
     display: inline-block;
     height: var(--terra-dialog-header-close-icon-height, 1.14286rem);
-    margin-top: var(--terra-dialog-header-close-icon-margin-top, -0.214285rem);
     vertical-align: middle;
     width: var(--terra-dialog-header-close-icon-width, 1.14286rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-dialog-header-close-icon-margin-top, 0.07rem);
```

Let's rename this variable to the following to denote its scoped to Microsoft vendor prefix
```suggestion
      margin-top: var(--terra-dialog-header-close-icon-margin-ms-top, 0.07rem);
```

---

Resolved in 2ff8a50